src/main/java/org/opendevstack/provision/controller/ProjectApiController.java

```diff
@@ -251,9 +251,6 @@ public ResponseEntity<Object> addProject(
       // create the delivery chain, including scm repos, and platform project
       newProject = createDeliveryChain(newProject);
 
```

Comment:
Maybe you should remove the method in addition to the removal of this call to it...
Is it possible to remove the method or it is used somewhere else?